Service/RequestModelArgumentResolver.php

```diff
@@ -45,6 +45,7 @@ public function __construct(Serializer $serializer, EventDispatcherInterface $di
      */
     public function supports(Request $request, ArgumentMetadata $argument)
     {
+        if (empty($argument->getType())) return false;
```

Comment:
please format this line with PSR recomendation https://www.php-fig.org/psr/psr-2/#51-if-elseif-else and i will be glad to accept this pr